advancedTodo/src/redux/todoAppComplete/todoApp.ui.js

```diff
@@ -61,8 +62,8 @@ class TodoApp extends Component {
 	}
 

```

Comment:
Variable name `value` can be more descriptive. Consider renaming it to `inputValue` or something similar to indicate what it holds.